src/vs/editor/contrib/find/common/findController.ts

```diff
@@ -308,6 +311,14 @@ export class CommonFindController extends Disposable implements editorCommon.IEd
 		}
 		return true;
 	}
+
+	public showGlobalBufferTerm(): string {
+		return this._clipboardService.readFindText();
```

Comment:
Once `optional` is adopted, some null/undefined checks should be performed here

Reply:
I'm also not sure why this method is called `show...`, it looks like a getter/reader